spec/requests/api/v1/preferences/show_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+describe 'GET api/v1/preferences/:id' do
+  subject { get api_v1_preference_path(preference_id), headers: auth_headers, as: :json }
+
+  let(:preference) { create(:preference) }
+  let(:preference_id) { preference.id }
+  let(:user) { create(:user) }
+
+  context 'when success' do
+    it 'returns http status 200' do
+      subject
```

Comment:
Aca creo que se podría agregar un `before { subject }` debajo del context, y de esa manera evitas escribir el `subject` en cada bloque `it`. Solo una sugerencia, no es bloqueante.

Reply:
Buenísimo.Es verdad es mejor así  se evita repetir código.